src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/__tests__/autocomplete.command.rff.test.ts

```diff
@@ -9,34 +9,34 @@
 import { setup } from './helpers';
 
 describe('autocomplete.suggest', () => {
-  describe('RRF', () => {
-    it('does not suggest RRF in the general list of commands', async () => {
+  describe('FUSE', () => {
+    it('does not suggest FUSE in the general list of commands', async () => {
       const { suggest } = await setup();
       const suggestedCommands = (await suggest('FROM index | /')).map((s) => s.text);
-      expect(suggestedCommands).not.toContain('RRF ');
+      expect(suggestedCommands).not.toContain('FUSE ');
     });
 
-    // ToDo:n Enable when RRF is on tech preview
-    it.skip('suggests RRF immediately after a FORK command', async () => {
+    // ToDo:n Enable when FUSE is on tech preview
```

Comment:
we ended up removing this restriction for FUSE - it does not have to always follow after FORK anymore.
up to you when you want to remove this